packages/mdc-chips/chip-set/foundation.js

```diff
@@ -83,8 +91,7 @@ class MDCChipSetFoundation extends MDCFoundation {
    * @return {!Element}
    */
   addChip(text, leadingIcon, trailingIcon) {
-    const chipEl = this.adapter_.createChipElement(text, leadingIcon, trailingIcon);
-    this.adapter_.appendChild(chipEl);
+    const chipEl = this.adapter_.appendChip(text, leadingIcon, trailingIcon);
```

Comment:
Do these still need to return the chip element? Would it make more sense to return the chip itself, or nothing at all?

Reply:
Oops, nevermind, forgot the component needs it.

Reply:
The foundation needs to return the chip element so the component can push it to its array of chip references.